week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr[2]; // complete this statement
```

Comment:
This should be based on the array size.
```suggestion
function last(arr) {
  return arr[arr.length - 1]; // complete this statement
}
```